### PR TITLE
Don't unset DP_STATUS_FREEZE

### PR DESCRIPTION
**Pull request by loganmc10**
See the discussion here: https://github.com/ata4/angrylion-rdp-plus/pull/41

The emulator should be handling setting/unsetting this bit, not the GFX plugin

### CODE REVIEW

File: src/RDP.cpp [hunk 1]
```diff
@@ -554,15 +554,15 @@ inline u32 READ_RDP_DATA(u32 address)
 void RDP_ProcessRDPList()
 {
 	if (ConfigOpen || dwnd().isResizeWindow()) {
-		dp_status &= ~0x0002;
+		//dp_status &= ~0x0002;
```
Comment (gonetz) [inline src/RDP.cpp]: Why not just remove this line?

File: src/RDP.cpp [hunk 2]
```diff
@@ -554,15 +554,15 @@ inline u32 READ_RDP_DATA(u32 address)
...
 		dp_start = dp_current = dp_end;
 		gDPFullSync();
 		return;
 	}
 
 	const u32 length = dp_end - dp_current;
 
-	dp_status &= ~0x0002;
+	//dp_status &= ~0x0002;
```
Comment (gonetz) [inline src/RDP.cpp]: Why not just remove this line?

**Comment by loganmc10** [comment]
I thought it would be useful for "historical" purposes since it's been there for so long, but it could be removed as well.

**Comment by gonetz** [comment]
Git will keep the history for us. I'll remove these lines.

**Comment by gonetz** [comment]
Merged: a2793ace2324

**Pull request closed without merging**